# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_net-tools.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- net-tools-1.60/lib/fddi.c.vanilla	2005-02-06 17:24:42.114093992 +0100
+++ net-tools-1.60/lib/fddi.c	2005-02-06 17:26:39.740212104 +0100
@@ -29,6 +29,7 @@
 #if __GLIBC__ >= 2
 #include <netinet/if_fddi.h>
 #else
+#include <asm/types.h>
 #include <linux/if_fddi.h>
 #endif
 #include <stdlib.h>
